.circleci/config.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - checkout
       - run:
           name: Install dependencies
-          command: apk update && apk add curl
+          command: apk add --update-cache --upgrade curl
```

Comment:
Fixes this dependency error when installing `curl` now on Alpine: https://app.circleci.com/pipelines/github/CircleCI-Public/server-terraform/573/workflows/291cfcc8-3794-4bde-9f23-bc2ecf70c7b9/jobs/2014

Reply:
TIL